suite-native/module-accounts-import/src/screens/AccountImportLoadingScreen.tsx

```diff
@@ -68,23 +71,38 @@ export const AccountImportLoadingScreen = ({
         };
 
         const getAccountInfo = async () => {
-            const fetchedAccountInfo = await TrezorConnect.getAccountInfo({
-                coin: networkSymbol,
-                descriptor: xpubAddress,
-                details: 'txs',
-            });
+            const [fetchedAccountInfo] = await Promise.all([
+                TrezorConnect.getAccountInfo({
+                    coin: networkSymbol,
+                    descriptor: xpubAddress,
+                    details: 'tokenBalances',
+                }),
+                dispatch(
+                    // @ts-expect-error not sure what is going on
+                    updateFiatRatesThunk({
+                        ticker: {
+                            symbol: networkSymbol,
+                        },
+                        rateType: 'current',
+                        fiatCurrency,
+                    }),
+                ),
+            ]);
 
             if (!ignore) {
                 if (fetchedAccountInfo?.success) {
                     if (networkSymbol === 'eth') {
                         fetchedAccountInfo.payload.tokens?.forEach(token => {
                             dispatch(
-                                updateCurrentFiatRatesThunk({
+                                // @ts-expect-error not sure what is going on
+                                updateFiatRatesThunk({
                                     ticker: {
-                                        symbol: token.symbol as string,
+                                        symbol: token.symbol as TokenSymbol,
```

Comment:
Couldn't you reuse `EthereumTokenSymbol` which is already available in `/suite-native`?

Reply:
We will merge these two types together. I can't use one from native because we will be moving fiat-rates to common. For now I just make alias from it in `suite-native/ethereum-tokens/src/types.ts` because I don't want to replace it in this PR because of too many changes.